flow/e2eshared/e2eshared.go

```diff
@@ -76,12 +76,7 @@ func ReadFileToBytes(path string) ([]byte, error) {
 	}
 	defer f.Close()
 
-	ret, err := io.ReadAll(f)
-	if err != nil {
-		return ret, fmt.Errorf("failed to read file: %w", err)
-	}
-
-	return ret, nil
+	return io.ReadAll(f)
```

Comment:
this pattern is so the error is "tagged" for logs, if all callsites of this function do it then this should be fine to change